### DISCUSSION TITLE
Is it possible to transfer management of a particular server to another Dokploy instance?

### DISCUSSION THREAD

**Discussion by pshemek** [Dokploy/dokploy#753] [Q&A]
I'm thinking about using the cloud, but it can be also a concern when the Dokploy instance is self-hosted.

Let's say I've got a Dokploy cloud account and manage like 8 servers there for different clients. Now, one of them resigns from my services (or vice versa). He owns the server but everything is managed from my Dokploy instance.
I would like to be able to set up another instance of Dokploy for him (cloud or self-hosted) and connect it to his server with all the apps running there (or even add Dokploy instance to that server).

How hard would it be? Is it even possible?
Do you consider it as a problem at all? I mean this strong "coupling" between the managing instance and the managed server with live production apps there.

Thanks.

**Comment by Siumauricio** [comment]
Hello, there would be no problem, that is if your client owns the server and changes the access keys (ssh), then you would lose control of that server regardless if you use the cloud or self hosted version, however the information would always be there persistent in your instance of dokploy, cloud or self hosted, so if in the future you want to reconnect that instance simply update the credentials in the part of multi server - edit, and your applications will be intact if you do not delete them.

**Reply by pshemek** [reply]
Thank you for the reply. Let me precise: whether it is possible to attach a different instance of Dokploy to such server which was previously managed by other instance.

Assume I'd be that client and I'd like to take the managament from someone who set up the server & apps for me via his Dokploy instance.

The parallel question could be, how usable (manageable) is such "standalone" server (without the origin Dokploy connected to it).

**Reply by Siumauricio** [reply]
No, it is not possible
